stock_outgoing_shipment_report/report/stock_outgoing_shipment_report.py

```diff
@@ -43,7 +43,7 @@ class StockOutgoingShipmentReport(models.TransientModel):
     partner_phone = fields.Char("Phone")
     product_code = fields.Char("Product Code")
     product_name = fields.Char("Product Name")
-    case_qty = fields.Float("Case Quantity")
+    case_qty = fields.Char("Case Quantity")
     separate_qty = fields.Char("Separate Quantity")
```

Comment:
I don't like making the quantity fields char type. Let me check with the customer on the presentation of the output.